src/site/pages/contribution.md

```diff
@@ -34,7 +34,7 @@ There are several properties available on the Example component to help with sty
 | ------------------- | ------------------------------------------------------------------- |
 | `heading`   | Sets the text displayed above the example (required) |
 | `minHeight`   | Sets the min-height of the preview box - useful for collapsed items that need space to expand or when a set minimum height would provide value (optional) |
-| `docs`   | Sets a reference to an example specific markdown file instead of documentation for the entire component to describe specific functionality displayed (optional) |
+| `docs`   | Sets a reference to an example specific markdown (.md) file to describe specific functionality displayed |
```

Comment:
Just wanted to follow up with:
I like Christie's suggestion.
Should this be "...an example-specific markdown (.md) file..." (i.e should the hyphen be used here)?

Reply:
👍